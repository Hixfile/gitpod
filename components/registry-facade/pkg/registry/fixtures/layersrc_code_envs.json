{
  "sourceRef": "eu.gcr.io/gitpod-core-dev/build/ide/code:akosyakov-enable-code-cli-2422.12",
  "content": {
    "678fadb4d3cfb2e238ea5a5d9dcd8239bb0b0809088ad13b9af63ee7c06cb72a": "eyJjcmVhdGVkIjoiMjAyMC0xMi0xOFQxMzo1Nzo1OS4wMzYzNjM2NDlaIiwiYXJjaGl0ZWN0dXJlIjoiYW1kNjQiLCJvcyI6ImxpbnV4IiwiY29uZmlnIjp7IkVudiI6WyJQQVRIPS91c3IvbG9jYWwvc2JpbjovdXNyL2xvY2FsL2JpbjovdXNyL3NiaW46L3Vzci9iaW46L3NiaW46L2JpbiIsIkdJVFBPRF9BUFBFTkRfRU5WX1BBVEg9L2lkZS9iaW46IiwiR0lUUE9EX1NFVF9FTlZfRURJVE9SPWNvZGUiLCJHSVRQT0RfU0VUX0VOVl9WSVNVQUw9Y29kZSIsIkdJVFBPRF9TRVRfRU5WX0dJVF9FRElUT1I9Y29kZSAtLXdhaXQiXX0sInJvb3RmcyI6eyJ0eXBlIjoibGF5ZXJzIiwiZGlmZl9pZHMiOlsic2hhMjU2OjhjYTQzNTZmNDRjYWUyOWU1MmFiMTY5ZTVmMjhhMDgwYzhjODhkM2RiODAzMjVlM2I2OGJkYTJmZDZmM2VlNGIiLCJzaGEyNTY6MDYyYWM4NDk3YzU0MWIyZDFiYTYxNzY4YTk4ZWRhMjU2ODUzYjI4Mzg5NTgxYzc5OTkxNTRlMjUzZmEzZmRmOSIsInNoYTI1NjozZjE3YmI4MmQ4NDczY2Q2MDFhZjA3MmE3MGQ3YjdjYTg0MTUwMWIwMjVhZjM4NDUzZTZhYjk5YjQwOGRjY2JiIiwic2hhMjU2OjYxODE3MTI5YWIwNDk0ZmYxODhhZDJkOTE0YzJlYjI1MDE3OTAwNmQzNjZmZWRlNjI0MTQ3ZTZhOGVhMzU3ZDQiLCJzaGEyNTY6NWYxNzBkMmY5Y2VlMTRmYmQyOThhMjUwNzQ1MDIxOTQwZWZkOTU0YTlhZjUwMjVlNDkxMmEzN2YzODZmN2NjYSJdfSwiaGlzdG9yeSI6W3siY3JlYXRlZCI6IjIwMjAtMTItMThUMTM6NTI6MzAuNTkwNzgyNjlaIiwiY3JlYXRlZF9ieSI6Ii9iaW4vc2ggLWMgIyhub3ApIENPUFkgZGlyOjgxYWJhNGZjNDQwOWIwMDkzYmMxNDg2ZDRiZTExNTZkNzNhYzFjODE5NGU2NjI4NzRiOTQ1MGZjNGRiNjZlYWMgaW4gL2lkZS8gIn0seyJjcmVhdGVkIjoiMjAyMC0xMi0xOFQxMzo1MjozMi41NTM4MDIyMDNaIiwiY3JlYXRlZF9ieSI6Ii9iaW4vc2ggLWMgIyhub3ApIENPUFkgZGlyOjMxZTJiMDEwMjRkMzRhMjdkNTgzNTUxM2FkNmQxMGE5MWNhMmY2MDEwZTM0OTU5MmIzN2JhOTE2YjA3OTc4NGEgaW4gL2lkZS8gIn0seyJjcmVhdGVkIjoiMjAyMC0xMi0xOFQxMzo1MjozNC44ODM0MzczNTRaIiwiY3JlYXRlZF9ieSI6Ii9iaW4vc2ggLWMgIyhub3ApIENPUFkgZGlyOjU4MDE2ZGU5YTQyMTBiYTg2MzNkMmRiYTAzMzFjYjJkMjA3ZWViMzc4ZTgyYzhkNjVhZjA0NGY4ZDQ1YTAyMTcgaW4gL2lkZS9ub2RlICJ9LHsiY3JlYXRlZCI6IjIwMjAtMTItMThUMTM6NTI6MzUuNjMzMjIwNDU0WiIsImNyZWF0ZWRfYnkiOiIvYmluL3NoIC1jICMobm9wKSBDT1BZIG11bHRpOjdjOTM5Mjc4MzcwNGNkNDI5NDYyOWQwMjhhZjU0NjNiZDRlNzE3OTkzNzQyNjYzNWQ0YmNmNTM0NTY2MzI3MzggaW4gL2lkZS8gIn0seyJjcmVhdGVkIjoiMjAyMC0xMi0xOFQxMzo1MjozNS43NzQwNDAxMjFaIiwiY3JlYXRlZF9ieSI6Ii9iaW4vc2ggLWMgIyhub3ApIENPUFkgZGlyOjQwYWU1ZDEwY2Q0ZDU4NjgyNmM1OGUyNTk1NWZiYjUyODZhYTUyNTdjMjQ3NTNmMWYyMGZjNGEyOGExYzUxM2IgaW4gL2lkZS9iaW4gIn0seyJjcmVhdGVkIjoiMjAyMC0xMi0xOFQxMzo1MjozNi4wMDU1MzkwODdaIiwiY3JlYXRlZF9ieSI6Ii9iaW4vc2ggLWMgIyhub3ApICBFTlYgR0lUUE9EX0FQUEVORF9FTlZfUEFUSD0vaWRlL2JpbjoiLCJlbXB0eV9sYXllciI6dHJ1ZX0seyJjcmVhdGVkIjoiMjAyMC0xMi0xOFQxMzo1MjozNi4yNTMxNjgxNDlaIiwiY3JlYXRlZF9ieSI6Ii9iaW4vc2ggLWMgIyhub3ApICBFTlYgR0lUUE9EX1NFVF9FTlZfRURJVE9SPWNvZGUiLCJlbXB0eV9sYXllciI6dHJ1ZX0seyJjcmVhdGVkIjoiMjAyMC0xMi0xOFQxMzo1Nzo1OC4zNDU4MjM2NzRaIiwiY3JlYXRlZF9ieSI6Ii9iaW4vc2ggLWMgIyhub3ApICBFTlYgR0lUUE9EX1NFVF9FTlZfVklTVUFMPWNvZGUiLCJlbXB0eV9sYXllciI6dHJ1ZX0seyJjcmVhdGVkIjoiMjAyMC0xMi0xOFQxMzo1Nzo1OS4wMzYzNjM2NDlaIiwiY3JlYXRlZF9ieSI6Ii9iaW4vc2ggLWMgIyhub3ApICBFTlYgR0lUUE9EX1NFVF9FTlZfR0lUX0VESVRPUj1jb2RlIC0td2FpdCIsImVtcHR5X2xheWVyIjp0cnVlfV19",
    "eu.gcr.io/gitpod-core-dev/build/ide/code:akosyakov-enable-code-cli-2422.12": "eyJtZWRpYVR5cGUiOiJhcHBsaWNhdGlvbi92bmQuZG9ja2VyLmRpc3RyaWJ1dGlvbi5tYW5pZmVzdC52Mitqc29uIiwiZGlnZXN0Ijoic2hhMjU2OmY4Y2U4ZWZkMDU2YWIyNDgxMzA0NWI1MzhmMmQxYTU1MTMzNzA3OTk1MzEwNmQyMmM5NThmMmYwNGNlYjQ0ODkiLCJzaXplIjoxMzY4fQ==",
    "f8ce8efd056ab24813045b538f2d1a551337079953106d22c958f2f04ceb4489": "eyJzY2hlbWFWZXJzaW9uIjoyLCJjb25maWciOnsibWVkaWFUeXBlIjoiYXBwbGljYXRpb24vdm5kLmRvY2tlci5jb250YWluZXIuaW1hZ2UudjEranNvbiIsImRpZ2VzdCI6InNoYTI1Njo2NzhmYWRiNGQzY2ZiMmUyMzhlYTVhNWQ5ZGNkODIzOWJiMGIwODA5MDg4YWQxM2I5YWY2M2VlN2MwNmNiNzJhIiwic2l6ZSI6MzEzMH0sImxheWVycyI6W3sibWVkaWFUeXBlIjoiYXBwbGljYXRpb24vdm5kLmRvY2tlci5pbWFnZS5yb290ZnMuZGlmZi50YXIuZ3ppcCIsImRpZ2VzdCI6InNoYTI1NjowMzgwNGUxYmUxYWU5N2JiYzU0YmZlZmNkZTI2YzJjODY5NmZjNDU0MTAzYTNhMzZlMDYyZWFmOTY0Yzc0YjQ4Iiwic2l6ZSI6MTIzMzgzMTl9LHsibWVkaWFUeXBlIjoiYXBwbGljYXRpb24vdm5kLmRvY2tlci5pbWFnZS5yb290ZnMuZGlmZi50YXIuZ3ppcCIsImRpZ2VzdCI6InNoYTI1Njo3N2VkNzc4NDljMGY5NzA1OTI2NjY1NmZlYzcxMGM5MGJlMDlhOTg0NzFiMjc4NmE4ZTViNjAzMmE5NmRkMjJkIiwic2l6ZSI6MjUwODE5MTB9LHsibWVkaWFUeXBlIjoiYXBwbGljYXRpb24vdm5kLmRvY2tlci5pbWFnZS5yb290ZnMuZGlmZi50YXIuZ3ppcCIsImRpZ2VzdCI6InNoYTI1Njo5YjZkNmI3OGNjYTQ5MjgzYjlhOWU3NGVjYTdlMzdjNjgzYTNlNmJhMTQwZGRjZjg5YTQyMDU2ZWYyZDQwM2I5Iiwic2l6ZSI6MjMyNjMyNjd9LHsibWVkaWFUeXBlIjoiYXBwbGljYXRpb24vdm5kLmRvY2tlci5pbWFnZS5yb290ZnMuZGlmZi50YXIuZ3ppcCIsImRpZ2VzdCI6InNoYTI1NjpmYTgzM2YzZGVmNjhkZGY3MmI3MmIyOWY5NmVlZDQ4MmI4NzE5NzIwZWE1NmEwYzFlMWVlMjQ5ODAwYzg5MTEwIiwic2l6ZSI6MTE5NX0seyJtZWRpYVR5cGUiOiJhcHBsaWNhdGlvbi92bmQuZG9ja2VyLmltYWdlLnJvb3Rmcy5kaWZmLnRhci5nemlwIiwiZGlnZXN0Ijoic2hhMjU2OjgyZDUxY2IxODA4ZWRiMjEwYTRjZjcyYjRkNzdiNTk0NDdjY2UwYmFjYWIzMWMwM2E5OWNlNmI5MjU3ZWEyZDEiLCJzaXplIjoyNzN9XX0="
  },
  "envs": [
    "PATH=/usr/bin",
    "EDITOR=vi",
    "GIT_EDITOR=vi"
  ]
}