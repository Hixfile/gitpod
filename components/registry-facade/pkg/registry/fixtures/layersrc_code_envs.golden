{
    "error": "",
    "layer": [
        {
            "Descriptor": {
                "mediaType": "application/vnd.docker.image.rootfs.diff.tar.gzip",
                "digest": "sha256:03804e1be1ae97bbc54bfefcde26c2c8696fc454103a3a36e062eaf964c74b48",
                "size": 12338319
            },
            "DiffID": "sha256:8ca4356f44cae29e52ab169e5f28a080c8c88d3db80325e3b68bda2fd6f3ee4b"
        },
        {
            "Descriptor": {
                "mediaType": "application/vnd.docker.image.rootfs.diff.tar.gzip",
                "digest": "sha256:77ed77849c0f97059266656fec710c90be09a98471b2786a8e5b6032a96dd22d",
                "size": 25081910
            },
            "DiffID": "sha256:062ac8497c541b2d1ba61768a98eda256853b28389581c7999154e253fa3fdf9"
        },
        {
            "Descriptor": {
                "mediaType": "application/vnd.docker.image.rootfs.diff.tar.gzip",
                "digest": "sha256:9b6d6b78cca49283b9a9e74eca7e37c683a3e6ba140ddcf89a42056ef2d403b9",
                "size": 23263267
            },
            "DiffID": "sha256:3f17bb82d8473cd601af072a70d7b7ca841501b025af38453e6ab99b408dccbb"
        },
        {
            "Descriptor": {
                "mediaType": "application/vnd.docker.image.rootfs.diff.tar.gzip",
                "digest": "sha256:fa833f3def68ddf72b72b29f96eed482b8719720ea56a0c1e1ee249800c89110",
                "size": 1195
            },
            "DiffID": "sha256:61817129ab0494ff188ad2d914c2eb250179006d366fede624147e6a8ea357d4"
        },
        {
            "Descriptor": {
                "mediaType": "application/vnd.docker.image.rootfs.diff.tar.gzip",
                "digest": "sha256:82d51cb1808edb210a4cf72b4d77b59447cce0bacab31c03a99ce6b9257ea2d1",
                "size": 273
            },
            "DiffID": "sha256:5f170d2f9cee14fbd298a250745021940efd954a9af5025e4912a37f386f7cca"
        }
    ],
    "envs": [
        "PATH=/ide/bin:/usr/bin",
        "EDITOR=code",
        "GIT_EDITOR=code --wait",
        "VISUAL=code"
    ]
}